decisions.md

```diff
@@ -18,18 +18,21 @@ It may not always be possible to reach a consensus. In that case, the project le
 
 ## Core Team Decisions
 
-Ultimately, the final decision on major questions is made by the Core Team. Within the Core Team, we use both consensus and voting to make decisions.
-
-Consensus is usually explicit within the Core Team, as opposed to lazy. For the kind of issues we expect the Core Team to resolve, it's important to hear from each member. The Chair is responsible for getting each member's view and determining whether there is a consensus.
+Because the Core Team generally makes major decisions, affecting multiple projects, _lazy consensus_ is normally not used. For the kind of issues we expect the Core Team to resolve, it's important to hear from each member. The Chair is responsible for getting each member's view and determining whether there is a consensus.
```

Comment:
I tried to clarify the need for hearing from everyone in most cases and the fact that we don't use lazy consensus very much in Core Team decisions.